tdrs-frontend/src/reducers/reports.js

```diff
@@ -21,21 +22,23 @@ export const fileUploadSections = [
   'Stratum Data',
 ]
 
-export const getUpdatedFiles = (
+export const getUpdatedFiles = ({
```

Comment:
By making this accept an object instead we can explicitly name arguments rather than relying on their position in a function call, which prevents some bugs if fields aren't sent in the right order.